helm_deploy/values-dev.yaml

```diff
@@ -11,8 +11,10 @@ generic-service:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: "applicationinsights.dev.json"
     HMPPS_AUTH_URL: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth"
     HMPPS_AUTH_TOKEN_URL: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth/oauth/token"
+    HMPPS_DELIUS_API_URL: "https://cvl-probation-mock.hmpps.service.justice.gov.uk"
```

Comment:
this is fine for now but we might be able to get away with using the real dev environment for these - we have 3 cases that are currently allocated and available for HDC and can probably get more